eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,67 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+    
+    <!-- TODO: Ignore needed until https://github.com/NuGet/Home/issues/11059 is addressed. -->
+    <UsagePattern IdentityGlob="Nuget.*/*" />
+    <UsagePattern IdentityGlob="Microsoft.Build.NuGetSdkResolver/*" />
+    
+    <!-- TODO: Ignore needed until vstest gets a source build leg. -->
+    <UsagePattern IdentityGlob="Microsoft.TestPlatform.*/*" />
+    
+    <!-- TODO: Ignore needed until razor merges main into the 17.6 branch -->
+    <UsagePattern IdentityGlob="Microsoft*Razor.*/*" />
+    
+    <!-- TODO: Figure out what to do about the netcoreapp ref packages (these are probably being pulled
+         in via implicit versions and net6 targeting projects (e.g. tests) -->
+    <UsagePattern IdentityGlob="Microsoft.*.App.Ref/*6.0*" />  
+    
+    <!-- TODO: SBRPs or updates -->
```

Comment:
What's the plan around these?  Is there a reason to not add the non net7.0 tfm based packages now before merging?

Reply:
Mainly I wanted to get this checked in and then whittle down the remaining SBRPs, rather than waiting for SBRP repo flow into here. Basically, stop any potential additional SBRPs being introduced in the meantime.

Reply:
I am onboard with that. I was just trying to get a sense of how you were thinking about this.